test/e2e/argo_server_test.go

```diff
@@ -568,7 +568,30 @@ func (s *ArgoServerSuite) TestCronWorkflowService() {
 	s.Run("List", func(t *testing.T) {
 		// make sure list options work correctly
 		s.Given(t).
-			CronWorkflow("@testdata/basic.yaml")
```

Comment:
ah - maybe this is the problem making tests flakey?

Reply:
Yup, this was it!

Reply:
good work